db/migrations/20250122190038_add_warning_status_and_sub_status_to_work_items.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex, Promise) {
+  return knex.schema.raw(`
+    ALTER TABLE "work_items"
+    DROP CONSTRAINT "work_items_status_check",
+    ADD CONSTRAINT "work_items_status_check"
+    CHECK (status IN ('ready', 'queued', 'running', 'successful', 'failed', 'canceled', 'warning')),
+    ADD COLUMN "sub_status" VARCHAR(255),
+    ADD CONSTRAINT "work_items_sub_status_check"
+    CHECK (sub_status IN (null, 'no-data'));
```

Comment:
I don't think we should have an enumeration on this field which I think we were calling "category" - we should allow any string and don't need this constraint.

Reply:
Done.